file-generating/ronn/ronn.sif.md

```diff
@@ -0,0 +1,81 @@
+ronn.sif(1) - ronn in a alpine container
```

Comment:
would uppercase Alpine and also Ronn: -> Ronn in an Alpine container . Also, is the **(1)** necessary? otherwise maybe it's better to be removed

Reply:
The `(1)` is not really necessary, but it shows the user that they can replace the `1` with other numbers for different manpage types, like `1` = `Executable programs or shell commands`, and `6` = `Games` (for game manuals), etc...